packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -195,6 +216,27 @@ class InspectorPreferencesController extends DisposableController
     setHoverEvalMode(hoverEvalModeEnabledValue == 'true');
   }
 
+  Future<void> _initDefaultInspectorDetailsView() async {
+    await _updateInspectorDetailsViewSelection();
```

Comment:
- `_updateInspectorDetailsViewSelection` sets `_defaultInspectorDetailsView.value` to be whatever was set in storage
- and then in this method, we then set the value in storage to be whatever is in `_defaultInspectorDetailsView.value`? 

Maybe I'm misunderstanding, but is setting the value in storage again necessary?

I'm also curious what happens if a value is saved in storage, and then the user updates the setting. Where does the new setting get set in storage?

Reply:
The _updateInspector... checks if anyvalue is already set in storage, if so it sets the current default to that one. Otherwise the default default is used. I guess default is a bit confusing here since it is a double default :P

Line 222 here is a listener that will update the value in storage any time the listener gets updated.

Reply:
Ah got it! Makes sense